projects/packages/my-jetpack/src/products/class-backup.php

```diff
@@ -351,7 +351,14 @@ public static function does_module_need_attention() {
 		// First check the status of Rewind for failure.
 		$rewind_state = self::get_state_from_wpcom();
 		if ( ! is_wp_error( $rewind_state ) ) {
-			if ( $rewind_state->state !== 'active' && $rewind_state->state !== 'provisioning' && $rewind_state->state !== 'awaiting_credentials' ) {
+			// Special case: 'unavailable' with 'site_new' reason is a normal provisioning state for brand new sites.
+			$is_new_site_provisioning = 'unavailable' === $rewind_state->state &&
+										'site_new' === ( $rewind_state->reason ?? '' );
+
+			if (
+				$rewind_state->state !== 'active' && $rewind_state->state !== 'provisioning' && $rewind_state->state !== 'awaiting_credentials' &&
```

Comment:
```suggestion
				false === in_array( $rewind_state->state, array( 'active', 'provisioning', 'awaiting_credentials' ) ) &&
```

Reply:
@macbre Thanks for the suggestion. What about this alternative instead?
```suggestion
				! in_array( $rewind_state->state, array( 'active', 'provisioning', 'awaiting_credentials' ), true ) &&
```

Reply:
Addressed in https://github.com/Automattic/jetpack/pull/46008/commits/3be2a807a5d5fbf70394ba1b92626b26e313f2b4